### ISSUE TITLE
I don't know if this plugin support personal notifications sent by the bot.

### ISSUE THREAD

**Issue by nikolaigauss** [alexkvak/teamcity-slack#118]
Apologies if this has been answered before, I've been looking through the closed issues but it didn't seem clear to me if this is a thing or not.

Currently we've the Slack plugin configured in some build configurations to notify certain channels if a build job doesn't work, this works flawless.

What I'm also trying to achieve is for people to be able to use the plugin so that the bot DM the user (and I'm not sure if we call this personal notifications because there are also personal builds).

I've configured the notifications like this:

![image](https://user-images.githubusercontent.com/4080093/107948525-e0f19c80-6f8b-11eb-86ea-a6d4ad80b5dc.png)

But I don't get any DM notifications from the app in Slack.

Hopefully all of that makes sense, if not let me know and I'll explain with more details.

Thanks!

**Comment by alexkvak** [comment]
Hi, @nikolaigauss 

It seems that you asking about TeamCity core feature [Slack Notifier](https://www.jetbrains.com/help/teamcity/notifications.html#Slack+Notifier)

This plugin has another way to configuration. Please read README carefully. And yes, this plugin can send personal notifications and mention commiters in channels.

**Comment by nikolaigauss** [comment]
Hey, thanks for the quick turnaround! So I've checked the Slack Notifier and that's what I want, however I was wondering if I could use the plugin for that use case as well.

I've checked what you said about private notifications, I think you're reffering to this:
![image](https://user-images.githubusercontent.com/4080093/107950348-7f7efd00-6f8e-11eb-8347-64cc2bfda7a4.png)

What I don't understand is how these notifications work, will it try to send a DM message to the users involved in the changes that managed to break the build?
